src/postgres/src/test/regress/yb_aggregates_schedule

```diff
@@ -0,0 +1,18 @@
+# src/test/regress/yb_pg_aggregates_schedule
+#
+####################################################################################################
+# This includes tests for Aggregates Functions
+####################################################################################################
+# Dependencies for `yb_pg_aggregates`
+test: yb_pg_int4
+test: yb_pg_int8
+test: yb_pg_create_aggregate
+
+# Chain of dependencies for `yb_pg_aggregates`
+test: yb_pg_create_function_1
+test: yb_pg_create_type
+test: yb_pg_create_table
+test: yb_pg_copy
```

Comment:
Please move `yb_pg_create_aggregate` back out to a separate section as in `yb_pg_misc_serial_schedule`.  The ordering should align with `serial_schedule`'s ordering because it has the least chance of chain dependency issues for now and the future.

Reply:
I've created `yb_pg_aggregates_schedule` that has the right ordering in the new pull request. Also, since `yb_pg_aggregates` will have its own schedule, I'll delete it in `yb_pg_misc_serial_schedule` when I add the other (effectively single) tests.

Reply:
See: https://github.com/yugabyte/yugabyte-db/pull/5263